introduction.md

```diff
@@ -1,2 +1,5 @@
 Hello dojo
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?

Reply:
create a "new_rules" branch on your "dojo_rules" repository and a create new pull request from "new_rules" back to "master".